src/gradio/GradioClient.cpp

```diff
@@ -197,7 +197,7 @@ OpResult GradioClient::uploadFileRequest(const juce::File& fileToUpload,
     auto postEndpoint = uploadEndpoint.withFileToUpload("files", fileToUpload, mimeType);
 
     auto options = juce::URL::InputStreamOptions(juce::URL::ParameterHandling::inPostData)
-                       // .withExtraHeaders("Accept: */*")
+                       .withExtraHeaders(createExtraHeaders())
```

Comment:
I think this line causes issues, which may be why it was commented out originally.